src/zjs_event.c

```diff
@@ -478,9 +467,8 @@ static void zjs_event_create_prototype() {
 void zjs_make_emitter(jerry_value_t obj, jerry_value_t prototype,
                       void *user_data, zjs_event_free free_cb)
 {
-    if (!zjs_event_initialized) {
-        zjs_event_create_prototype();
-    }
+    zjs_event_create_prototype();
```

Comment:
Since the function name is "create", it should create, not "create unless something" - so this was better before with the if here I think.

Reply:
zjs_event_create_prototype already checks if its initialized, and just returns if it is already done.  No reason to check twice. I can name it something else if you'd prefer, but I think the check should live in the prototype function, so that if more code is added that calls it, we don't accidentally create another copy.

Reply:
I'm suggesting that you move the check back out where it was before. I doubt anyone else is going to call that. It doesn't make sense to call a function "create prototype" if you don't need to create the prototype and it's not actually going to create the prototype. It's wise to make the behavior match the English. :)

Reply:
Or if you insist, then a better name for the function would be ensure_prototype_exists().